apps/opik-backend/src/main/resources/liquibase/db-app-state/migrations/000003_add_last_created_experiment_at_to_datasets_table.sql

```diff
@@ -0,0 +1,6 @@
+--liquibase formatted sql
+--changeset thiagohora:add_last_created_experiment_at_to_datasets_table
+
+ALTER TABLE datasets ADD COLUMN last_created_experiment_at TIMESTAMP(6) DEFAULT NULL;
```

Comment:
Minor: with idempotency would be great e.g: ALTER TABLE IF EXITS... and ADD COLUMN IF NOT EXISTS. But I believe not supported by MySQL.

Reply:
Yeah, unfortunately, MySQL doesn't support it yet. Only `MariaDB` does :( 

REF: https://dev.mysql.com/doc/refman/8.4/en/alter-table.html